docker/scripts/dev_start.sh

```diff
@@ -21,8 +21,8 @@ FAST_BUILD_MODE="no"
 FAST_TEST_MODE="no"
 VERSION=""
 ARCH=$(uname -m)
-VERSION_X86_64="dev-18.04-x86_64-20200422_2150"
-VERSION_AARCH64="dev-aarch64-20170927_1111"
+VERSION_X86_64="dev-18.04-x86_64-latest"
+VERSION_AARCH64="dev-aarch64-latest"
```

Comment:
I'd prefer keeping exact tag, so that the image version and code version are well matched. Otherwise history versions of code would always break.

Reply:
> I'd prefer keeping exact tag, so that the image version and code version are well matched. Otherwise history versions of code would always break.

Good point!